### PR TITLE
Add Login and Register screens

### LINKED ISSUE #4: User can login (back end) S2
References:
[User login](https://guides.codepath.org/android/Building-Data-driven-Apps-with-Parse#user-session-login) 

Acceptance criteria:
- [x]  User can use a previously created account to login on login/sign up screen
Note: This ticket depends on log in/sign up screen

### LINKED ISSUE #5: Login/ Register screen  S1 S2
References: 
image

Acceptance criteria:
- [x] Screen should match wireframe 
- [x] Sign up should work (depends on #3 )
- [x] Login should work (depends on #4 )

### PR DESCRIPTION
**Pull request by kat-kime**
Close #5 
Close #4 

- Adds .xml file for Login screen
- Adds .xml file for Register screen
- Adds String resources
- Adds Login Functionality

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
+def apikeyProperties = new Properties()
+apikeyProperties.load(new FileInputStream(apikeyPropertiesFile))
+
 android {
     compileSdkVersion 30
     buildToolsVersion "30.0.1"
 
     defaultConfig {
+        // should correspond to key/value pairs inside the file
+        buildConfigField("String", "CLIENT_KEY", apikeyProperties['CLIENT_KEY'])
```
Comment (GavinSlusher) [inline app/build.gradle:14]: See note in ParseApplication.java

File: app/src/main/java/com/example/fond/ParseApplication.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+public class ParseApplication extends Application {
+
+    // Initializes Parse SDK as soon as the application is created
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        // TODO: Figure out why receiving an occasional "com.parse.ParseRequest$ParseRequestException: i/o failure" error
+        Parse.initialize(new Parse.Configuration.Builder(this)
+                .applicationId("BGw3ZXQSjmfO7mIwrcZ2FNNnytM8Ch0AeOM9Xnys")
```
Comment (GavinSlusher) [inline app/src/main/java/com/example/fond/ParseApplication.java]: Shouldn't this come from the apikey.properties file if we're going to have the file?

**Review by GavinSlusher — CHANGES_REQUESTED** [review]
I'm not sure why, but for I too get that error: `Issue with login: com.parse.ParseRequest$ParseRequestException: i/o failure` every time I attempt to log in.

I saw that you added the `usesCleartestTraffic` line to the android manifest like Codepath seemed to recommend. Does this implementation let you log in every time?

EDIT: It actually works perfect on the Pixel 2 API 27. Still not sure what the root cause is, but we probably should either look into it, or make sure that it definitely works for whatever phone we're choosing to develop off of.

EDIT2:

After restarting my computer, the login seems to work just fine. Odd that it's buggy like that. I still think we should move the parse API credentials out of ParseApplication.java because we all have different application IDs that we can leave in our apikey.properties file. Other than that, I think we're good to go!

File: app/src/main/java/com/example/fond/ParseApplication.java (hunk 2 above)
Comment (kat-kime) [reply app/src/main/java/com/example/fond/ParseApplication.java]: Ah, yes - I forgot to remove these during my panic of trying to figure out why the Parse server wasn't connecting. Just hid the keys now.

File: app/build.gradle (hunk 1 above)
Comment (kat-kime) [reply app/build.gradle:14]: Addressed.

**Review by GavinSlusher — APPROVED** [review]

**Pull request merged**